tests/integration/test_vulnerability_detector/test_feeds/test_download_feeds.py

```diff
@@ -96,7 +97,7 @@ def test_download_feeds(configuration, metadata, set_wazuh_configuration_vdt, tr
         - Check in log that the database provider has been updated successfully.
         - Check that the timestamp of the feed metadata does not exceed the established threshold limit.
 
-    wazuh_min_version: 4.4.0
+    wazuh_min_version: 4.7.0
```

Comment:
Let's avoid changing the version.

Reply:
Done in commit: docs: Undo changes in wazuh_min_version and modify Copyright in some tests